code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -7,6 +7,7 @@
 /mob/living/simple_animal/hostile/blob
 	icon = 'icons/mob/blob.dmi'
 	pass_flags = PASSBLOB
+	status_flags = 0 //No throwing blobspores into deep space to despawn, or throwing blobbernaughts, which are bigger than you.
```

Comment:
Can use the define `None` here to be a bit more verbose. It is the same as 0 but it reads a bit better

Reply:
Done!